index.js

```diff
@@ -214,23 +214,24 @@ SteamTradeOffers.prototype.cancelOffer = function(options, callback) {
   });
 };
 
-SteamTradeOffers.prototype.acceptOffer = function(options, callback) {
+SteamTradeOffers.prototype.acceptOffer = function(offer, callback) {
```

Comment:
The reason you wouldn't need to change the code if another issue arises is that it is likely the offer will contain any additional property that needs to be added later if needed, where that property would have to be explicitly added to the options.

If you would like to continue following the convention for this module, I can make that change though.